### ISSUE TITLE
paper #565 "Baital: An Adaptive Weighted Sampling Approach for Improved t-wise Coverage"

### ISSUE THREAD

**Issue by kstolee** [researchart/fse20#101]
This issue is for discussion related to PR #27 

Author contact: @edbaranov

**Comment by kstolee** [comment]
## Authors

| Author  | Email | Github ID |
| ------- | ----- | --------- |
| Eduard Baranov | eduard.baranov@uclouvain.be (corresponding) | edbaranov |
| Axel Legay | axel.legay@uclouvain.be | axelucl | 
| Kuldeep S. Meel | meel@comp.nus.edu.sg | kuldeepmeel |

**Comment by art-reviewer** [comment]
[Paper](https://github.com/edbaranov/fse20/blob/4bb6e29c4de0dcb8bc9501a7de6ebdb3f9815527/submissions/reusable/baital/Baital:%20An%20Adaptive%20Weighted%20Sampling%20Approach%20for%20Improved%20t-wise%20Coverage.pdf)

**Comment by anon-eval** [comment]
@edbaranov
Thank you for submitting the artifacts and congratulations on the paper :) 


**TLDR**: Based on the current version of your submission, I **recommend**  the reusable badge but I would recommend the following changes.
1. Please update your artifact on [doi.org](doi.org) and provide the DOI in `ReadMe.md`. 

The artifacts were verified on an Ubuntu 16.04 instance with 16GB RAM

I arrived at this conclusion using the following metrics
1. **Installation** ✅ : The installation runs seamlessly without any errors. Thanks also for the steps to verify the installation.
2. **Public Availability**🟡 : The tool is publicly available on github. I would recommend the artifact be uploaded on [doi.org](doi.org)
3. **Documentation**  ✅ : The `Readme` is well documented for reproducing the results and running the tool.
4. **Consistent/Complete**✅ : The results are consistent on multiple executions and all the results in the paper can be reproduced using the scripts provided.
5. **Exercisable** ✅  : The authors clearly mention how to run `Baital` and the scripts successfully run. 
6. **Evidence of verification and validation**✅ : The authors also provide documentation on how to reproduce the results in their study and results match the results in their paper.
7. **Reusability**✅ : The authors also mention how the tool can be reused. The authors also provide sufficient configuration files which can be reused and used to create new configurations for custom datasets.

Thanks :)

**Comment by gousiosg** [comment]
@anon-eval excellent review, thanks!
@seartifact We would appreciate your review till Sunday EoB.

**Comment by seartifact** [comment]
Hi Authors,

Overall the documentation you provided goes into details in terms of installation, set-up the environment, and running the tool. 

## Available
The artifacts is uploaded to Zenodo.

## Functional
The installation of packages ran perfectly well. I could also run the experiments as the scripts were provided by the authors.


## Reusable
The quality of the documentation is good. Authors provide enough information to reproduce as well as create new experiments. Certain spot checks of the results on multiple executions  reproduced the results mentioned in the paper (using the scripts provided).

**Comment by edbaranov** [comment]
Dear reviewers,

We would like to thank you for your reviews and feedback.

@anon-eval  The artifact is uploaded to Zenodo. We have updated the readme.md in the public github repository with the doi information. The readme.md in this repository (fse20) already include the doi.  Please let us know if we need to make other changes.

Thank you,
Authors

**Comment by kstolee** [comment]
Great! I'll tentatively apply the ```available``` and ```reusable``` badges as recommended. 

@edbaranov make sure you cite the artifact (using the DOI) in the References section of the paper.
